ReactWindows/ReactNative.Shared/ReactNative.Shared.projitems

```diff
@@ -162,6 +162,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\BlurEvent.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\FocusEvent.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\KeyEvent.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)UIManager\ImportantForAccessibility.cs" />
```

Comment:
Any reason to put this in shared if it's currently only being used for UWP?

Reply:
This enum we also want to use for implementation of accessibilityLabel and I thought that it may be usable not only for UWP but WPF too. But I will talk to Yury who plans to work on accessibilityLabel implementation and if he thinks the implementation will be limited to UWP only, then I'll move it ReactNative

---
In reply to: [164202844](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164202844) [](ancestors = 164202844)

Reply:
Moved to ReactNative

---
In reply to: [164528524](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164528524) [](ancestors = 164528524,164202844)